### DISCUSSION TITLE
first person to reply will be marked as answer | 🧠 Galaxy brain 🧠

### DISCUSSION THREAD

**Discussion by RealMrHex** [Schweinepriester/github-profile-achievements#584] [Q&A]
@schambig

**Comment by schambig** [comment]
@RealMrHex Thanks for your time, Let's get that badge! 🧠

**Comment by LauraBigoni** [comment]
For galaxy brain you need to suggest answers to questions posted in discussion. If your answer is helpful then the person who asked the question can mark you answer as the solution. Once you've had 2 questions marked as solutions then you'll get galaxy brain.

Here's the link to the general GitHub community discussions. It might be a good place to start. https://github.com/community

If you found this useful then please mark my answer as a solution so I can get the badge.
